WordPress/Classes/Stores/StatsInsightsStore.swift

```diff
@@ -981,3 +985,66 @@ private extension InsightStoreState {
     }
 
 }
+
+// MARK: - iOS 14 Widgets Data
+private extension InsightStoreState {
+
+    private func saveHomeWidgetTodayData(stats: TodayWidgetStats) {
```

Comment:
Did you consider having this as another function on `TodayWidgetStats` alongside `saveData`?  It feels like this is putting extra responsibilities on the Store, and I'm not sure if it would be better placed elsewhere?

Reply:
So... I don't necessarily feel like `TodayWidgetStats` is a better place to handle the saving because my (very personal) understanding of the responsibilities of a Store, is that it should take care of abstracting the rest of the code from where data is stored.

But let's try to get some value out of this thought... this made me think that maybe what we can improve here is to clearly specify who owns this particular responsibility in our class descriptions, whether it's the store, or `TodayWidgetStats`, and put all the relevant code together.

Reply:
Thanks for being so thorough on this. A few considerations:
- there's a TODO in that method, so it might change
- it's probably a bit confusing because of the coexistence of old and new Today Widgets; `TodayWidgetStats` would not be the right place for methods related to the new widgets, `HomeWidgetTodayData` would.
- This method takes the updated info from the store and then updates the Widget accordingly. It's the equivalent of the existing `storeTodayWidgetData` and, for that reason, I realize that the name should change to `storeHomeWidgetTodayData`, so that I'm gonna do for now.